langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingResponse.java

```diff
@@ -0,0 +1,30 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.model.bedrock.internal.BedrockEmbeddingResponse;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+/**
+ * Bedrock Cohere embedding response
+ */
+@Getter
+@Setter
+public class BedrockCohereEmbeddingResponse implements BedrockEmbeddingResponse {
```

Comment:
```suggestion
class BedrockCohereEmbeddingResponse implements BedrockEmbeddingResponse {
```
Does it have to be public?

Reply:
Removed Lombok and removed public